src/utils/urls.js

```diff
@@ -1,23 +1,24 @@
-export const homePath = '/';
-export const docsPath = '/docs';
-export const usersPath = '/users';
-export const settingsPath = '/settings';
-export const importBatchesPath = '/import-batches';
-export const createImportPath = '/import-batches/create';
-export const loginPath = '/login';
-export const logoutPath = '/logout';
-export const mySpacePath = '/my-space';
-export const registerPath = '/register';
-export const resetPasswordPath = '/reset-password';
-export const searchPath = '/search';
-
-export const docsPrefix = '/docs/';
-export const editDocPrefix = '/edit/doc/';
-export const revisionPrefix = '/revs/';
-export const pluginApiPathPrefix = '/plugins/';
-export const completeRegistrationPrefix = '/complete-registration/';
-export const completePasswordResetPrefix = '/complete-password-reset/';
-export const confirmRoomMembershipPrefix = '/confirm-room-membership/';
+const homePath = '/';
+const docsPath = '/docs';
+const usersPath = '/users';
+const loginPath = '/login';
+const logoutPath = '/logout';
+const searchPath = '/search';
+const mySpacePath = '/my-space';
+const registerPath = '/register';
+const settingsPath = '/settings';
+const importBatchesPath = '/import-batches';
+const resetPasswordPath = '/reset-password';
+const createImportPath = '/import-batches/create';
+
+const docsPrefix = '/docs/';
+const revisionPrefix = '/revs/';
+const editDocPrefix = '/edit/doc/';
+const roomDetailsPrefix = '/rooms/';
+const pluginApiPathPrefix = '/plugins/';
+const completeRegistrationPrefix = '/complete-registration/';
+const completePasswordResetPrefix = '/complete-password-reset/';
+const confirmRoomMembershipPrefix = '/confirm-room-membership/';
```

Comment:
Nice, soon we can enable the "don't double-export" eslint rule!

Reply:
No clue how far we are from that rule yet, but one day maybe 🤞